ONSdigital/rm-tools
# Add 2019 collection exercises

# Motivation and Context
Acceptance tests & CI pipeline broken. I had added new collection exercise dates for 2019 but forgot to add the actual collection exercises themselves.

# What has changed
Added 2019 collection exercises.

# How to test?
Hack acceptance test setup to pull from this branch and run `make setup`. Check that 2019 collexes are set up correctly.

# Links
Trello: https://trello.com/c/DOsrckW2/585-bug-fix-the-ci-pipeline-failure-due-to-collection-exercise-go-live-date
